lckernel/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ cad/operations/builder.cpp
 cad/primitive/customentity.cpp
 cad/interface/snapconstrain.cpp
 cad/dochelpers/documentlist.cpp
+../lcUI/widgets/settings.cpp
```

Comment:
This line should be moved to lcUI/CMakeLists.txt

Reply:
It is there in lcUI/CMakeLists.txt but if I remove it from here I wont be able to use Settings::getVal() in const.h

Reply:
You can move the Settings class in the kernel, as it doesn't add new dependencies